From 22e1bd138dbcf32ca50f9659c9d847fdae11a5f0 Mon Sep 17 00:00:00 2001
From: Zac Medico <zmedico@gentoo.org>
Date: Sun, 9 Apr 2017 14:54:17 -0700
Subject: pm_plugins/portage: drop portage._native_kwargs usage (bug 613936)

The portage._native_kwargs function is not available since
portage-2.3.0.

X-Gentoo-Bug: 613936
X-Gentoo-Bug-URL: https://bugs.gentoo.org/613936
---
 pm_plugins/portage/sync/modules/laymansync/subproc.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

(limited to 'pm_plugins/portage/sync/modules/laymansync/subproc.py')

diff --git a/pm_plugins/portage/sync/modules/laymansync/subproc.py b/pm_plugins/portage/sync/modules/laymansync/subproc.py
index 1166841..477bb46 100644
--- a/pm_plugins/portage/sync/modules/laymansync/subproc.py
+++ b/pm_plugins/portage/sync/modules/laymansync/subproc.py
@@ -72,7 +72,7 @@ class Layman(NewBase):
 
         exitcode = portage.process.spawn_bash("%(command)s" % \
             ({'command': command}),
-            **portage._native_kwargs(self.spawn_kwargs))
+            **self.spawn_kwargs)
         if exitcode != os.EX_OK:
             msg = "!!! layman add error in %(repo)s"\
                 % ({'repo': self.repo.name})
@@ -111,7 +111,7 @@ class Layman(NewBase):
         command = ' '.join(args)
         exitcode = portage.process.spawn_bash("%(command)s" % \
             ({'command': command}),
-            **portage._native_kwargs(self.spawn_kwargs))
+            **self.spawn_kwargs)
 
         if exitcode != os.EX_OK:
             exitcode = self.new()[0]
-- 
cgit v1.2.1

